storybookjs/storybook
# App for Mithril

Issue:

## What I did

Add a new storybook app for [Mithril](https://mithril.js.org/) framework

## How to test

Is this testable with jest or storyshots?

- I think so, but I'm not sure.

Does this need a new example in the kitchen sink apps?

- I included a new example in this PR.

Does this need an update to the documentation?

- Maybe, yes.

If your answer is yes to any of these, please make sure to include it in your PR.

- Please tell me documents which I should modify. (I'm not a native English speaker.)
